pkg/cpi/mockingUtils.go

```diff
@@ -78,3 +80,50 @@ func GetNegativeCaseHTTPResponseBodyAndErrorNil() (*http.Response, error) {
 	}
 	return &res, nil
 }
+
+//GetIntegrationArtifactGetMplStatusCommandMockResponse -Provide http respose body
+func GetIntegrationArtifactGetMplStatusCommandMockResponse(testType string) (*http.Response, error) {
```

Comment:
well, again, one gets more than the response, a nil error is also returned. To be compliant with the previous steps as well I suggest to add that to the function name.

Reply:
I was wrong. In the negative case an error **is** returned.